overlay/lower/usr/sbin/irled

```diff
@@ -33,7 +33,7 @@ case "$mode" in
 		gpio clear $pin > /dev/null
 		;;
 	read)
-		cat /sys/class/gpio/gpio${pin}/value
```

Comment:
the pin you are queryig may be not exported. say a led pin which are defined in the environment but not exported by default. so you will be doing cat from a non-extent file, resulting in an error.

Reply:
Yep, new lesson learned. I think this patch should cure the problem, but let me know if you foresee other gotchas.

Reply:
Now it will work. Merging. Thank you.

Reply:
Thank you! This project is fantastic!